src/containers/home/index.tsx

```diff
@@ -29,6 +34,11 @@ export const Homepage = () => {
       console.error(error);
     }
   };
+  useEffect(() => {
+    if (isUnverifiedSession(sessionData) && !isVerifiedSession(sessionData)) {
```

Comment:
Because of the way predicates are named, I assume they are the opposite of each other thus this operation is weird. Maybe create a predicate that only returns true if and only if the user is undefined and use that